demo/lightning/text-based/README.md

```diff
@@ -0,0 +1,57 @@
+# Parquet Demo Code
+The code examples in this directory demonstrate how dataflux can be used for text-based training alongside Pytorch-Lightning. To support this training, this example subclasses DataFluxIterableDataset for compatibility with parquet files from the HuggingFace FineWeb dataset.
+
+Please note that these demos are for educational and example purposes only, and have not been optimized for performance.
+
+## Single Node
+The single node exmaple code here can be run on your local workstation with a command such as the following:
+
+```
+# python3 ./model.py --project=test-project --bucket=my-fineweb-data --batch-size=100 --prefix="fineweb/sample/10BT"
```

Comment:
Should the path to model.py include the directory here, assuming the user is in the same directory as the readme?